packages/ui/src/components/Button/index.tsx

```diff
@@ -20,17 +22,21 @@ export function Button({
   className = '',
   children,
   onClick,
-  size,
+  size = 'medium',
+  fullWidth = false,
+  icon,
 }: ButtonProps) {
+  const block = fullWidth ? 'block' : '';
```

Comment:
here you want to try this https://styled-components.com/docs/basics#adapting-based-on-props

repeating this one here

Reply:
Yes, yes. I haven't forgot about that one. I actually read about that one but I avoid it for some other issues that appear,  I am thinking how to solve them this time. But thanks for repeating it. It is better to have it here, easier to see!

Reply:
Hi @paschalidi,
Hi man, this issue is driving me crazy, Every solution I tried throws a new problem. I am assuming we want to use [this](https://styled-components.com/docs/basics#adapting-based-on-props) technique directly in Storybook. I have try different solutions but I cannot solve it. 

What would the right step me to follow now? If I ask for your help should I present what I have tried? It is a lot of code to past here and I know that is not right. Should I push different commits with different failed solutions? Could you point me to the right way of acting in a situation like this one? I know we have had similar situations, but I am trying to get a feeling for what to do in each case.

Reply:
usually in a working scenario you would pair with someone for 10 ~ 20 minutes. In this situation what I can do is create a https://codesandbox.io/s/pass-in-props-sr258 and I will try to create something similar there.

Reply:
hi @paschalidi ,
Thanks for that man! I wasn't at home and I saw you answered something and I couldn't wait to get here to see what you did. This was the line I couldn't figure out:

`<ButtonStyles isPink={isPink}>`

Now it is so obvious!!!! :D Thanks man